tests/pytest/test_runner.py

```diff
@@ -48,6 +48,8 @@ def test_runner(parameters):
         vhdl_sources = [os.path.join(tests_dir, "designs", "runner", "runner.vhdl")]
         gpi_interfaces = [vhdl_gpi_interfaces]
 
+    pre_cmd = ["echo everything is fine!;"]
```

Comment:
Is there any way to automatically verify this message is printed?

Reply:
For 1, that's fine. For 2, it really doesn't matter; we use `grep` in other tests, so I'm happy with that.

Reply:
Actually, `grep` not really good idea for parsing output, because `echo` will be printed twice: once as a command `echo blabla;` and second time as a result of command: `blabla`.

So, I've removed pre_cmd from Riviera flow and made test to cover possible pre_cmd cases for Questa.